### PR TITLE
docs(argo-workflows): Add missing value on values.yaml

### PR DESCRIPTION
**Pull request by yu-croco**
Signed-off-by: yu-croco <yuki.kita22@gmail.com>

In this PR https://github.com/argoproj/argo-helm/pull/769 , `fullnameOverride` was added but it was not defined on `values.yaml`, so I added it. 🙋 

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [x] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

File: charts/argo-workflows/values.yaml [hunk 1]
```diff
@@ -12,6 +12,10 @@ init:
 
 createAggregateRoles: true
 
+# Override app name
+# Internally, this will be truncated at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+fullnameOverride: ""
+
```
Comment (yu-croco) [inline charts/argo-workflows/values.yaml:18]: This is used below. 🙋 
https://github.com/argoproj/argo-helm/blob/master/charts/argo-workflows/templates/_helpers.tpl#L24-L39

Comment (mkilchhofer) [inline charts/argo-workflows/values.yaml:18]:
```suggestion
## String to partially override "argo-workflows.fullname" template
##
# nameOverride:

## String to fully override "argo-workflows.fullname" template
##
# fullnameOverride:
```

Comment (yu-croco) [reply charts/argo-workflows/values.yaml:18]: Thank you for reviewing! I agree with it.

> Also nameOverride was missing. WDYT?

I completely forgot it... 😇 You are right.

I will fix as you suggested.

Comment (yu-croco) [reply charts/argo-workflows/values.yaml]: I fixed on this commit 🙋 
https://github.com/argoproj/argo-helm/pull/782/commits/81e2e5d7708196997a5454394e0a86d11595c427

File: charts/argo-workflows/values.yaml [hunk 2]
```diff
@@ -12,6 +12,13 @@ init:
 
 createAggregateRoles: true
 
+## String to partially override "argo-workflows.fullname" template
+##
+# nameOverride:
+## String to fully override "argo-workflows.fullname" template
```
Comment (mkilchhofer) [inline charts/argo-workflows/values.yaml:18]:
```suggestion

## String to fully override "argo-workflows.fullname" template
```

File: charts/argo-workflows/values.yaml [hunk 3]
```diff
@@ -12,6 +12,14 @@ init:
 
 createAggregateRoles: true
 
+## String to partially override "argo-workflows.fullname" template
+##
+# nameOverride:
```
Comment (mkilchhofer) [inline charts/argo-workflows/values.yaml:17]: I think, we'd better use this without being a comment, so tools like ArgoCD detect this value in the UI :-) Sorry for that, as it was my suggestion to use it like this.
```suggestion
nameOverride:
```

File: charts/argo-workflows/values.yaml [hunk 4]
```diff
@@ -12,6 +12,14 @@ init:
...
+
+## String to fully override "argo-workflows.fullname" template
+##
+# fullnameOverride:
```
Comment (mkilchhofer) [inline charts/argo-workflows/values.yaml:21]: I think, we'd better use this without being a comment, so tools like ArgoCD detect this value in the UI :-) Sorry for that, as it was my suggestion to use it like this.
```suggestion
fullnameOverride:
```

**Review by mkilchhofer — APPROVED** [review]

**Pull request merged**